src/amo/components/AddonMoreInfo/index.js

```diff
@@ -247,9 +256,10 @@ export class AddonMoreInfoBase extends React.Component<Props> {
   }
 }
 
-export const mapStateToProps = (state: Object) => {
+export const mapStateToProps = (state: {| user: UserStateType |}) => {
   return {
     userId: state.user.id,
+    canViewStats: isAdmin(state) || hasPermission(state, STATS_VIEW),
```

Comment:
Since the concept of `isAddonAuthor()` is checked separately, it might be a useful clarification to name this property more explicitly like `hasStatsPermission`.

Reply:
That's what I had before and then changed because of `isAdmin()`.